CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing to Office UI Fabric
+
+## Creating New Components
+
+### Creating a component via the command line
+1. Open a command prompt in the root directory of your project directory.
+2. Run `npm run createcomponent -- --name YourAwesomeNewComponentName`
+  * Our components use the Pascal Case naming convention.
+
+### Creating a component manually.
+1. From the root of the project, travel to `packages/office-ui-fabric-react/src/components/`
```

Comment:
need to refine this documentation to explain the process of adding new components.

For new ideas:
create components in `experiments`
consume in products, refine

To prep to move into oufr:
get design approval
add documentation
add tests
add vr tests
migrate to oufr